### ISSUE TITLE
Requirements

### ISSUE THREAD

**Issue by rhalder2023** [cyrusbehr/YOLOv8-TensorRT-CPP#66]
So i did read the requirements for cuda and Tensorrt versions, can i not make it work with cuda 11.8 and tensorrt 8.6??

**Comment by rpribau** [comment]
Yes, you can make it work using CUDA 11.8 and TensorRT 8.6! I have it working on my local machine running for some projects I'm currently have. The only thing is please, run it on Ubuntu/Linux.

**Comment by rhalder2023** [comment]
> Yes, you can make it work using CUDA 11.8 and TensorRT 8.6! I have it working on my local machine running for some projects I'm currently have. The only thing is please, run it on Ubuntu/Linux.

yeah i wouldnt dream of using a some other OS
